hp_search_benchmarks/nas_search_spaces/darts/cnn/model_def.py

```diff
@@ -0,0 +1,180 @@
+"""
+This model is from the CNN NAS search space considered in:
+    https://openreview.net/forum?id=S1eYHoC5FX
+
+We will use the adaptive searcher in Determined to find a
+good architecture in this search space for CIFAR-10.  
+"""
+
+import sys
+from collections import namedtuple
+from typing import Any, Dict
+
+import torch
+from torch import nn
+import torchvision.datasets as dset
+
+import determined as det
+from determined.pytorch import DataLoader, LRScheduler, PyTorchTrial, reset_parameters
+
+# These imports will be from the darts/cnn module.
+# The darts repo will be cloned in the startup-hooks.sh script.
+sys.path.append("./darts/cnn")
+from model import NetworkCIFAR as Network
```

Comment:
We typically discourage sys path hacking in our examples -- is there a way that `from darts.cnn.model import ...` will work?

Reply:
i'll probably have to iteratively add __init__.py files to the original repo.  do you think that's better than adding it to the path?

Reply:
Got it. In this case the best option might be to fork https://github.com/quark0/darts to `determined-ai` and add a commit fix it up into a proper Python module.